plugins/woocommerce/src/Internal/Admin/ImportExport/HposOrderExportHandler.php

```
@@ -0,0 +1,244 @@
+		}
+	}
+
+	/**
+	 * Outputs a single order in WXR-compatible XML format.
+	 *
+	 * @param \WC_Order $order The WooCommerce order object.
+	 */
+	protected function export_order_to_xml( $order ) {
+		$order_id       = $order->get_id();
+		$date_created   = $order->get_date_created() ? $order->get_date_created()->format( 'Y-m-d H:i:s' ) : '';
+		$date_modified  = $order->get_date_modified() ? $order->get_date_modified()->format( 'Y-m-d H:i:s' ) : '';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$date_modified'. (Unused Code Rules)

(UnusedLocalVariable)